test/object-store/realm.cpp

```diff
@@ -428,9 +428,9 @@ TEST_CASE("SharedRealm: get_shared_realm()") {
         REQUIRE(realm4->schema().find("object") != realm4->schema().end());
     }
 
-// The ExternalCommitHelper implementation on Windows doesn't rely on files
-#ifndef _WIN32
```

Comment:
It's fine for this entire test to be disabled and not show up at all on Windows - it's testing named pipes as filesystem entities, and on Windows named pipes aren't files, so it doesn't apply.